File: Task7/ex4.py
using namespace std;

int main() {
  int n, i;
    
  cin >> n;
  i=0;
  while (i < n) 
    {
        i++;
        if (i<n) cout << i << " "; 
        else cout << i << endl; 
    }  
  while (i > 0) 
    {
        if (i>1) cout << i << " ";
        else cout << i;
        i--;
    }  
}

File: 7class/Tema10/ex4.cpp
using namespace std;

int main() {

  int a, b, i, max, nod;
  cin >> a >> b;
  nod=1;
  if (a>b) max=a; else max=b;
  for (i=2; i<=max; i++)
   {
      if (a%i==0 && b%i==0) nod=i;
   }
  cout << nod;
}